### PR TITLE
5111 - neworder page

### PR DESCRIPTION
**Pull request by alibombaye**
Screenshot 2020-05-07 at 10 35 16

[AB#6764](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6764)

Just the page with title and description for this task. 
Buttons to be included in next task

### CODE REVIEW

File: app/pages/order-dashboard/controller.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { getContext } from './contextCreator';
+
+export const getNewOrderPageContext = () => getContext({ pageName: 'neworder' });
```
Comment (kat1906) [inline app/pages/order-dashboard/controller.js:3]: is the `pageName` here so you can reuse this component?

Comment (alibombaye) [reply app/pages/order-dashboard/controller.js:3]: yep. we'll be using the same template for existing order dashboard page

File: app/routes.js [hunk 2]
```diff
@@ -36,7 +37,8 @@ export const routes = (authProvider) => {
   }));
 
   router.get('/organisation/neworder', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
-    res.status(200).send('new order page');
+    const context = getNewOrderPageContext();
+    res.render('pages/order-dashboard/template.njk', addContext({ context, user: req.user }));
```
Comment (kat1906) [inline app/routes.js:41]: is it the order dashboard page or is it the task list page? (too many dashboards)

Comment (alibombaye) [reply app/routes.js:41]: Glad you brought this up. I wasn't too comfortable with the name. I agree too many dashboards is getting confusing. specially when both dashboards are around orders

Shall we go for `order-tasks-list` instead?

Comment (kat1906) [reply app/routes.js:41]: `order-task-list`

Comment (kat1906) [reply app/routes.js:41]: lol yeah

File: app/pages/order-task-list/neworder/manifest.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "backLinkText": "Back",
```
Comment (kat1906) [inline app/pages/order-task-list/neworder/manifest.json:2]: `Go back`

Comment (alibombaye) [reply app/pages/order-task-list/neworder/manifest.json:2]: DOH!!!

**Pull request merged**